lib/row.rb

```diff
@@ -0,0 +1,125 @@
+require 'field_serializer'
+require 'nokogiri'
+
+class String
+  def tidy
+    gsub(/[[:space:]]+/, ' ').strip
+  end
+end
+
+class Row
+  include FieldSerializer
+
+  def initialize(tds)
+    @tds = tds
+  end
+
+  field :id do
+    tds[0].xpath('a/@title')
+          .text
+          .downcase
+          .gsub(/ /,'_')
+          .gsub('_(page_does_not_exist)','')
+  end
+
+  field :name do
+    tds[0].xpath('a').text.tidy
+  end
+
+  field :coalition do
+    return if coalition_node.nil?
+    coalition_node.xpath('@title').text
+  end
+
+  field :coalition_id do
+    return if coalition_node.nil?
+    coalition_node.text
+  end
+
+  field :party do
+    return if party_node.nil?
+    party_node.xpath('@title').text
+  end
+
+  field :party_id do
+    return if party_node.nil?
+    party_node.text
+  end
+
+  field :state do
+    tds.xpath('./preceding::h3[1]')
+       .text
+       .gsub('[edit]', '')
+       .tidy
+  end
+
+  field :wikipedia do
+    wiki_link
```

Comment:
Why do you need a separate method for this, rather than just having that method be this field?